src/cronParser.ts

```diff
@@ -25,13 +25,43 @@ export class CronParser {
    * @returns {string[]}
    */
   parse(): string[] {
-    let parsed = this.extractParts(this.expression);
+    let parsed: string[];
+    
+    var expression = this.expression ?? '';
+
+    if (expression.startsWith('@')) {
+      var special = this.parseSpecial(this.expression);
+      parsed = this.extractParts(special);
+
+    } else {
+      parsed = this.extractParts(this.expression);
+    }
+
     this.normalize(parsed);
     this.validate(parsed);
 
     return parsed;
   }
 
+  parseSpecial(expression: string): string {
+    const specialExpressions: { [key: string]: string } = {
+        '@yearly': '0 0 1 1 *',
+        '@annually': '0 0 1 1 *',
+        '@monthly': '0 0 1 * *',
+        '@weekly': '0 0 * * 0',
+        '@daily': '0 0 * * *',
+        '@midnight': '0 0 * * *',
```

Comment:
`@midnight` - I'm not aware of a cron implementation that supports this.  Is there one?

Reply:
yep see here: https://www.unix.com/man-page/linux/5/crontab/